pkg/odo/cli/application/delete.go

```diff
@@ -62,7 +63,7 @@ func (o *DeleteOptions) Run() (err error) {
 		}
 		log.Infof("Deleted application: %s from project: %v", o.appName, o.Project)
 	} else {
-		log.Infof("Aborting deletion of application: %v", o.appName)
+		return fmt.Errorf("Aborting deletion of application: %v", o.appName)
```

Comment:
shouldn't this be info? user intervention should be an error?

Reply:
Updated